week-2/14-align-self/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Flexbox</title>
+  <link rel="stylesheet" href="/css/normalize.css">
+  <link rel="stylesheet" href="/css/week2.css">
+	<link rel="stylesheet" href="/css/week2-flexbox.css">
+  <link rel="stylesheet" href="flexbox.css">
+  <link rel="stylesheet" href="./ignore_this_folder/custom.css">
+</head>
+
+<body>
+  <div class="site-wrapper site-wrapper">
+    <div class="site-header">
+      <h1 class="site-header__title">First</h1>
+    </div>
+    <ul class="countries countries--tall countries--first">
+			<li class="country" id="brazil">Brazil</li>
```

Comment:
Let's not use the `id` attribute here. For one, there are multiple instances of the same `id` on the page, which I don't think is valid. But also, let's reinforce the BEM naming structure we've been using, and use class names like `country--brazil`.

Reply:
yes, very good spot, wasn't intentional to use multiple ids with the same name